traffic_incident_parser/src/traffic_incident_parser_worker.cpp

```diff
@@ -174,7 +179,31 @@ namespace traffic
       path_point.y=i.y();
       traffic_mobility_msg.geometry.nodes.push_back(path_point);
     }
-      return traffic_mobility_msg;
+
+    std::vector<cav_msgs::TrafficControlMessageV01> output_msg;
+
+    traffic_mobility_msg.params.detail.choice=cav_msgs::TrafficControlDetail::CLOSED_CHOICE;
+    
+    if(event_type=="CLOSED")
+    {
+    traffic_mobility_msg.params.detail.closed=cav_msgs::TrafficControlDetail::CLOSED;
+    }
+    else if(event_type=="OPEN")
+    {
+    traffic_mobility_msg.params.detail.closed=cav_msgs::TrafficControlDetail::OPEN;
+    }
+    traffic_mobility_msg.package.label=event_reason;
```

Comment:
event_reason only filled out when the event_type is closed. Otherwise, package label shall set as empty string.